pygotham/frontend/sponsors.py

```diff
@@ -73,7 +73,7 @@ def edit(pk):
     return render_template('sponsors/edit.html', form=form, sponsor=sponsor)
 
 
-@route(blueprint, '/')
+@route(blueprint, '/', navbar_kwargs={'path': ('Sponsors', 'Sponsors')})
```

Comment:
I definitely think this should be renamed, but I think that's out of scope of this PR. The renaming should probably happen in one swoop once we have our new navbar hierarchy laid out.